options/label/Advanced.yaml

```diff
@@ -0,0 +1,62 @@
+labels:
```

Comment:
why yaml

the original (contrib/options/label/Advanced) use the same format as options/label/Default

Reply:
To be able to add additional options, like label priority:
https://github.com/go-gitea/gitea/pull/11669/files#diff-f00731f73124774aa19202aff69b10c8R50

Reply:
we should convert options/label/Default then too ?

Reply:
No need, both ways are supported to keep backward compatibility